examples/mobile/UIComponents/components/notification-badge/number-badge-appbar.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../lib/tau/mobile/js/tau.js"></script>
+</head>
+
+<body>
+	<div class="ui-page">
+		<header>
+			<div class="ui-appbar-left-icons-container">
+				<a href="#" class="ui-btn ui-btn-icon ui-btn-icon-back" data-style="flat" data-rel="back"></a>
+			</div>
+			<div class="ui-appbar-title-container">
+				<span class="ui-appbar-title">Sample title</span>
+			</div>
+			<div class="ui-appbar-action-buttons-container">
+				<button class="ui-btn ui-btn-icon ui-btn-icon-more" data-style="flat" onClick="openDropdownMenu()">
+					<span class="ui-badge" title="2"></span>
+				</button>
+				<select data-native-menu="false" id="sample-dropdown-menu" style="display: none">
+					<option value="1">
+						Option1
+					</option>
+					<option value="2" class="ui-badge" title="2">
+						Option2
+					</option>
+					<option value="3">
+						Option3
+					</option>
+					<option value="4">
+						Option4
+					</option>
+				</select>
+			</div>
+		</header>
+
+		<div class="ui-content">
+		</div>
+
+		<script>
+			var dropDownMenu = document.getElementById("sample-dropdown-menu");
```

Comment:
Why don't you move the content to a js file?

Reply:
I'll fix it.